modules/virt-about-monitored-resources-openshift-virtualization.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-about-monitored-resources-openshift-virtualization.adoc
+//
+
+[id="virt-about-monitored-resources-openshift-virtualization_{context}"]
+= About monitored resources in {VirtProductName} Overview
+
+The following table shows actively monitored resources in the *{VirtProductName} Overview* page. This information is useful when you need obtain relevant data and intervene to resolve a problem.
+
+[cols="1,1"]
+|===
+|*Monitored resource information* | *Description*
+|Details
+|A brief overview of service and version information for *{VirtProductName}*.
+|Status
+|Alerts for virtualization, storage, and networking.
```

Comment:
Storage isn't part of the MVP for 4.10. Currently, only virtualization and networking are part of the Status card.

Reply:
Removed storage.